audobject/core/api.py

```diff
@@ -0,0 +1,108 @@
+import os
+import typing
+
+import oyaml as yaml
+
+from audobject.core.object import Object
+import audobject.core.utils as utils
+
+
+def _decode_value(
```

Comment:
Usually, we move the private functions to the bottom of the file.

Reply:
> In my opinion we could shorten the function names to:
> 
> ```python
> audobject.from_*
> ```

Agree. Changed it accordingly.